iptables/iptables.go

```diff
@@ -2,12 +2,19 @@ package iptables
 
 import (
 	"errors"
+	"net"
 
 	"github.com/coreos/go-iptables/iptables"
 )
 
 // AddRule adds the required rule to the host's nat table
 func AddRule(appPort, metadataAddress, hostInterface, hostIP string) error {
+
+	err := InterfaceExists(hostInterface)
```

Comment:
Nit: could do `if err := InterfaceExists(hostInterface); err != nil`...

Not a big deal tho :)